src/30_Advanced/Advanced_Video_Docking.html

```diff
@@ -2,16 +2,30 @@
   "hidePreview": true,
   "experiments": ["video-dock"]
```

Comment:
One thing I forgot: can you enable preview mode by adding: `"preview": "default"`.  Maybe also add a link to the preview in the introduction:

```
You can see the sample in action [here](preview).
```

Reply:
Preview template breaks the demo:

![screen shot 2018-09-27 at 2 39 44 pm](https://user-images.githubusercontent.com/254946/46176153-416ccb80-c263-11e8-9d2c-730f00d21124.png)

Is there any way to force it to full-width?

Reply:
add:

```
main {
   width: 100%;
}
```

Preview is on my to fix list for amp.dev. 

Another thing, it looks a bit broken on mobile:

![screenshot 2018-09-27 at 15 12 36](https://user-images.githubusercontent.com/380472/46177543-1042ca00-c268-11e8-8245-02a7dccd584f.png)